SportHub/src/modules/SearchResultsList/helpers/getNumberOfArticlesRequest.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios"
+
+const getNumberOfArticlesRequest = async (language, findText) => {
+    try {
+        const backendHost = process.env.REACT_APP_BACKEND_HOST
+        const response = await axios.get(`${backendHost}/Article/GetNumberOfSearchArticles?language=${language}&findText=${findText}`, {})
+        console.log(response.data)
+        return response
+    } catch (error) {
+        console.error(error)
+        return error.code
```

Comment:
you return error code but you didn't handle it in component

Reply:
![image](https://github.com/XashDM/SportHub/assets/113111483/c2ee69a0-4cca-4149-b030-c691598b8e77)
handling, but in one 'try-catch-finally' with getSearchArticlesRequest.